File: iniciante/python/1017-gasto-de-combustivel.py
def combustivel_gasto(tempo_gasto, velocidade_media):
    # automóvel faz 12 Km/L
    gasto_em_litro = tempo_gasto * velocidade_media * (1/12)
    return print(f'{gasto_em_litro:.3f}')


tempo = int(input())
velocidade = int(input())
combustivel_gasto(tempo, velocidade)

File: iniciante/c/1017-gasto-de-combustivel.c
#include <stdio.h>

int main(){
    float tempo_gasto, velocidade_media, litros;
    scanf("%f %f", &tempo_gasto, &velocidade_media);
    litros = velocidade_media * tempo_gasto / 12;
    printf("%.3f\n", litros);
    return 0;
}
